### PR TITLE
Update ref for RDF Term

### PR DESCRIPTION
**Pull request by afs**
Broken link.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/sparql-results-xml/pull/26.html" title="Last updated on May 3, 2023, 11:03 AM UTC (5f17f0b)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/sparql-results-xml/26/54335a6...5f17f0b.html" title="Last updated on May 3, 2023, 11:03 AM UTC (5f17f0b)">Diff</a>

### CODE REVIEW

**Review by rubensworks — APPROVED** [review]

File: spec/index.html [hunk 1]
```diff
@@ -462,7 +462,7 @@ <h4>Variable Binding Results</h4>
 &lt;/sparql&gt;
 </pre>
   <p>Each <code>result</code> element corresponds to one <em>Query Solution</em> in a result and contains child elements (in no particular order) for each <em>Query Variable</em> that appears in the
-  solution. It is used to record how the query variables bind to <a data-cite="SPARQL12-QUERY#defn_RDFTerm">RDF Term</a>s.</p>
+  solution. It is used to record how the query variables bind to <a data-cite="RDF12-CONCEPTS#dfn-rdf-term">RDF Term</a>.</p>
```
Comment (TallTed) [inline spec/index.html:465]:
```suggestion
  solution. It is used to record how a query variable is bound to an <a data-cite="RDF12-CONCEPTS#dfn-rdf-term">RDF Term</a>.</p>
```

Comment (TallTed) [reply spec/index.html:465]: or
```suggestion
  solution. It is used to record how query variables bind to <a data-cite="RDF12-CONCEPTS#dfn-rdf-term">RDF Terms</a>.</p>
```

**Comment by TallTed** [comment]
"Broken link" doesn't include "breaking plural/singular consistency".

**Comment by TallTed** [comment]
Which I now see was fixed in #27, without connecting back here...

**Comment by TallTed** [comment]
> As you know, the main branch is protected. That means PRs can not be changed after merge to main.

True. It's too bad this PR was merged so quickly — a mere 33 minutes after creation! — giving other eyes so little time to review and advise except via another PR, which really shouldn't necessary to fix such a typo within a PR.

**Comment by afs** [comment]
It was a link fix for FPWD. It was approved.

**Comment by TallTed** [comment]
It started as a link fix. Then it added an editorial bit by editing the text atop that link, which introduced a plural/singular conflict in context.

I imagine the approval was based on looking at the URL change, and not at the text atop it. My eyes gravitated elsewhere, upon first look.

**Pull request merged**